test/use-cases.hs

```diff
@@ -7,14 +7,29 @@
 -- the 3-clause BSD licence.
 --
 
+{-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies      #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
 
 module Main where
 
 import           Network.URI
-import           System.Environment     (getArgs, withArgs)
-import           System.FilePath.Posix  (splitFileName)
-import           Test.Hspec.WebDriver
+import           System.Environment          (getArgs, withArgs)
+import           System.FilePath.Posix       (splitFileName)
+import           Test.Hspec.WebDriver        hiding (BrowserDefaults (..))
+import           Test.WebDriver.Capabilities
+
+instance Using Browser where
```

Comment:
http://hackage.haskell.org/package/hspec-webdriver-1.0.2/docs/Test-Hspec-WebDriver.html#t:BrowserDefaults

"To obtain more control over the capabilities (e.g. to test multiple versions of IE or to test Firefrox without javascript), you should import Test.Hspec.WebDriver hiding (BrowserDefaults) and then create your own enumeration which is an instance of TestCapabilities and Using."